### ISSUE TITLE
Remove `data-raw` folder and `data/{(bench_regions, index_regions, currencies)}

### ISSUE THREAD

**Issue by jdhoffa** [2DegreesInvesting/PACTA_analysis#636]
These files are passed directly as data inputs now. 
A quick search all indicates that `bench_regions` and `index_regions` don't seem to be used anywhere. 

`currencies` does seem to be, however it should likely always be pulled from the input data?

**Comment by cjyetman** [comment]
thanks... I will give this some thought

**Comment by jdhoffa** [comment]
I think there are a lot of files in `data` that are seemingly unused... I don't know if they are used by CONSTRUCTIVA somewhere down the line, but it could be interesting to assess which ones are actually necessary.

**Comment by cjyetman** [comment]
yeah, I already pruned a lot of the obvious stuff, but some this I have to think hard about, because like you said it could be expected by Constructiva or something in the Docker image build

**Comment by jdhoffa** [comment]
Yup totally agree. All that said, it's definitely worth us knowing if it IS or ISN't (otherwise CONSTRUCTIVA may end up using dated currency info for example).
